src/store/index.js

```diff
@@ -187,19 +186,19 @@ const store = {
     retry_message ({getters}, payload) {
       const currentChat = getters.getCurrentChat
       const partner = currentChat.partner
-      const message = currentChat.messages[payload]
-      let messageText = message.message.replace(/<\/?p>/g, '')
+      let message = currentChat.messages[payload]
+      let messageText = message.message
+      messageText = messageText.replace(/<p>|<\/?p>/g, '')
+      messageText = messageText.replace(/<br>/g, '\n')
       payload = {
         recipientId: partner,
-        message: messageText,
+        message: message.message,
```

Comment:
why here is message.message? it should be either messageText or code above doesn't make sense